### ISSUE TITLE
Currently  1.14.8.52 version is working

### ISSUE THREAD

**Issue by half0522** [Soreepeong/XivAlexander#548]
I deleted the 8.54 version and reinstalled it with 8.52 version and it is currently working fine with 6.3 opcodes.
Try it

**Comment by thiamaster** [comment]
fake news. Crash as soon as you execute an action.

**Comment by Tsukalos** [comment]
Can confirm that 8.52 is working with the current opcodes

**Comment by seungxseung** [comment]
what are the opcodes for 6.3?

**Comment by Defy559** [comment]
I did this yesterday when someone mentioned it and it doesn't work. It still gives errors over and over. Yes alex will boot with the game but that's it.

**Comment by Tsukalos** [comment]
Can confirm, game doesn't crash, but it logs a `Data error`:
```
[1969-12-31 21:00:00.000] Length=64 ConnType=1 Count=1 Gzip=2
2023-01-11 16:26:59.084	SocketHook	Error: Data error
```

**Comment by half0522** [comment]
I used this code https://github.com/Soreepeong/XivAlexander/commit/2786e1096b898de28556948dc98ec6b8f3fa4af9

**Comment by Aang07** [comment]
> Hey it seems to be working for me, is there a way for me to provide a log or something?

You should have a big window open while using Alexander, that's updating non-stop. That's the logs. 
If not, try looking under some menu options on the Alexander window. 
I can't open it, so I can see exactly where to go to show it.

**Comment by brendonpagano** [comment]
Just a FYI: I would not go around testing different versions of Alex to see if it works, specially with different op codes. You're just generating weird logs on your machine and sending it over to the game servers. In a more crucial situation, it could even  get you banned or put your account flagged under suspicious activity.

**Comment by CalmWeaver27** [comment]
This is false. My game crashed 4 times while attempting this. DO NOT use version 8.52

**Comment by ThewickedA-913** [comment]
First time when i reinstalled .52 version, game crashed again. But i tried to reinstall FFXIV and .52 version, then it's perfectly working. Just delete and reinstalling FFXIV and try again.

**Comment by YongeI6** [comment]
I uninstalled .54 and reinstalled .52 as the op and it works fine, no crash encountered. Might be other software issues for people who have their game crashed I assume. The 6.3 opcode is working, my game is smoother than without alex, still, it feels clunky to press the buttons, I would hope an actual update come out soon.

**Comment by exaflareenjoyer** [comment]
false. not working. game crashes.
